<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyForm</class>
 <widget class="QWidget" name="MyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLCDNumber" name="m_plcd"/>
     </item>
     <item>
      <widget class="QSlider" name="m_psld">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="m_pcmdReset">
       <property name="text">
        <string>&amp;Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pcmdQuit">
       <property name="text">
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>m_psld</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_plcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pcmdQuit</sender>
   <signal>clicked()</signal>
   <receiver>MyForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
